web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.tsx

```diff
@@ -7,6 +12,7 @@ import { AnalysisTable, CreateAnalysisButton } from './components'
 import { Divider } from '@equinor/eds-core-react'
 
 export const AnalysisOverview = (): JSX.Element => {
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore